app/predacons_cli/src/cli.py

```
@@ -0,0 +1,45 @@
+
+
+class Cli:
+    def __init__(self):
+        self.predacons = predacons
+
+    def launch(self):
+        # check if the predacon_cli_config.json file exists
+        if not Cli.check_config_file():
+            Cli.create_config_file()
+        else:
+            config = Cli.load_config_file()
```

Ruff

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)